tests/E2E/combine-reports.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+import uuid
+import glob
+import json
+from datetime import datetime, timedelta
+
+
+def gt(dt_str):
```

Comment:
the naming of and in this function are not really good, would you mind improving them?

Reply:
I'm currently rewriting it, it's just a POC doing in the train :roll_eyes: